playwright/src/test/java/com/microsoft/playwright/TestPageEvaluate.java

```diff
@@ -532,16 +532,15 @@ void shouldWorkWithCSP() {
 
   @Test
   void shouldEvaluateException() {
-    try {
+    PlaywrightException e = assertThrows(PlaywrightException.class, () -> {
       page.evaluate("() => {\n" +
         "  return (function functionOnStack() {\n" +
         "    return new Error('error message');\n" +
```

Comment:
```suggestion
        "    throw new Error('error message');\n" +
```
This should fix the test.